tokio/src/task/mod.rs

```diff
@@ -232,6 +232,11 @@ cfg_rt_core! {
     pub use self::join::JoinHandle;
 }
 
+cfg_rt_threaded! {
```

Comment:
I would consider `rt-util` or `task-util`. This feature could depend on `rt-core` as it implies having a runtime.